src/main/java/com/codecool/dao/sql/ExpLevelSQL.java

```diff
@@ -14,26 +20,146 @@ public ExpLevelSQL(BasicConnectionPool connectionPool) {
 
     @Override
     public void addExpLevel(ExpLevel expLevel) {
+        try {
+            Connection connection = connectionPool.getConnection();
+            addExpLevelToDatabase(expLevel, connection);
+            connectionPool.releaseConnection(connection);
+        } catch (SQLException e) {
+            System.err.println("SQLException: " + e.getMessage()
+                    + "\nSQLState: " + e.getSQLState()
+                    + "\nVendorError: " + e.getErrorCode());
+        }
+    }
+
+    private void addExpLevelToDatabase(ExpLevel expLevel, Connection connection) throws SQLException {
+        try (PreparedStatement stmtInsertExpLevelData = connection.prepareStatement(
+                "INSERT INTO experience_levels(name, experience_amount) VALUES(?, ?)")) {
+            insertExpLevel(stmtInsertExpLevelData, expLevel);
+        }
+    }
 
+    private void insertExpLevel(PreparedStatement stmtInsertExpLevelData, ExpLevel expLevel) throws SQLException {
+        stmtInsertExpLevelData.setString(1, expLevel.getName());
+        stmtInsertExpLevelData.setString(2, expLevel.getName());
```

Comment:
For second time you call for expLevel.getName where you should call for expLevel.getExpAmount

Reply:
Fixed